### PR TITLE
initial version, ui project supported

### PR DESCRIPTION
**Pull request by VaclavK**
I will be talking to @ruimserrano re some aspects of the template pipeline itself but that is not related to the code itself

### CODE REVIEW

**Comment by VaclavK** [comment]
@djfr @dguerin NuGet is up and running, 1.0.143 succeeded(https://eshopworld.visualstudio.com/Github%20build/_build/results?buildId=35362&view=logs) and it will eventually reflect here

File: src/Sierra.Actor/BuildDefinitionActor.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@
     using Microsoft.TeamFoundation.Build.WebApi;
     using Model;
 
-    [StatePersistence(StatePersistence.Persisted)]
```
Comment (dguerin) [inline src/Sierra.Actor/BuildDefinitionActor.cs]: @VaclavK what's the need to change this? (just for context)

Comment (VaclavK) [reply src/Sierra.Actor/BuildDefinitionActor.cs]: consistency... the actors have volatile state - the true state is in DB

Comment (dguerin) [reply src/Sierra.Actor/BuildDefinitionActor.cs]: thanks man, yeah I can see it's not actually needed here for the build job at hand.

**Comment by djfr** [comment]
We will have to think about the "project type" -> to repo assignment and how we are going to address this, I don't think we can rely on Tahoe to tell us this, but we're better off thinking about this later on then now!

**Comment by VaclavK** [comment]
>           We will have to think about the "project type" -> to repo assignment and how we are going to address this, I don't think we can rely on Tahoe to tell us this, but we're better off thinking about this later on then now!

agreed @djfr , for API vs UI we "can" deduce ourselves and the checks would be quite simple - either based on the build pipeline for the master repo or through master repo source check

**Pull request merged**